CONTRIBUTING.md

```diff
@@ -5,3 +5,69 @@ We love improvements to our tools! EDGI has general [guidelines for contributing
 ## Submitting Web Monitoring Issues
 
 Issues that are project-wide, or relate heavily to the interaction between different components, should be added to our [Web Monitoring issue queue](https://github.com/edgi-govdata-archiving/web-monitoring/issues). Component-specific issues should be added to their respective repository.
+
+The following are recommended code styling and best practices for the web-monitoring-ui repository. We also have best practices for related to all the [web-monitoring] project (https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md) repo.
```

Comment:
Can you add a heading here? Otherwise this looks like part of the section on submitting issues:

<img width="701" alt="screen shot 2017-12-28 at 1 02 25 pm" src="https://user-images.githubusercontent.com/74178/34423252-970f2b16-ebcf-11e7-81b0-e913df60c58d.png">

Reply:
Added Heading